onnxruntime/core/providers/cpu/tensor/gather_nd.cc

```diff
@@ -63,12 +71,20 @@ Status GatherNDBase::PrepareForCompute(OpKernelContext* context, Prepare& p) con
     element_counts[i] = input_shape.SizeFromDimension(i + 1);
   }
 
+  auto last_dim_size = indices_shape.SizeFromDimension(indices_shape.NumDimensions() - 1);
```

Comment:
is it same as "auto last_dim_size = indice_shape[indice_shape.NumDimensions() - 1]"?

Reply:
I will make the change in another PR.